Core/Core/Models/ConversionResult.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace Speckle.Core.Models;
+
+public sealed class ConversionResult
+{
+  public string? ResultId { get; }
+  public object? Result { get; }
+  public Exception? Error { get; }
+  public object Target { get; }
+
+  //[MemberNotNullWhen(true, nameof(Result))]
+  //[MemberNotNullWhen(true, nameof(ResultId))]
+  //[MemberNotNullWhen(false, nameof(Error))]
+  public bool IsSuccessful => Result is not null;
+
+  public ConversionResult(object target, object result, string resultId)
+  {
+    Target = target;
+    Result = result;
```

Comment:
This object might know whether the conversion is successful with fallback displayValue or not. I mean, it is "successful" with "but". TBD

Reply:
Potentially simply the result type will be enough for us to communicate this if we wish.
"Brep" was converted as "Mesh" communicates exactly the same thing as "with dispalyValues", but imo will be more intuitive to the user.

If we wanted more than this, I could see some utility in knowing what converter was used to convert the object. 
But this would require some more restructuring of our "...WithFallBack" service, because right now, the converter is abstracted and the connector doesn't see it.

I think we could do something nice, where the "...WithFallBack" simply returns the converter (without calling it), instead of right now, where it both finds and immediately calls it